lib/readline.js

```diff
@@ -47,9 +47,7 @@ function Interface(input, output, completer, terminal) {
   }
   historySize = historySize || kHistorySize;
 
-  completer = completer || function() { return []; };
-
-  if (typeof completer !== 'function') {
+  if (completer && typeof completer !== 'function') {
```

Comment:
By allowing falsy values, we keep backwards compatibility. They were accepted before this.